SteamKit2/Tests/StreamHelpersFacts.cs

```diff
@@ -36,46 +36,44 @@ public void IsThreadSafe()
             }
         }
 
-        ConcurrentBag<Exception> threadExceptions = new ConcurrentBag<Exception>();
+        ConcurrentBag<Exception> threadExceptions = [];
```

Comment:
I'm surprised the target typing works with this, I would have expected `= new();`

Reply:
It works and VS automatically converts to this.

Reply:
interesting. both produce the same IL anyway so seems fine, just surprising.